### ISSUE TITLE
`make_co_wd` doesn't run in the test suite, but other test cases rely on it for starting models.

### ISSUE THREAD

**Issue by adamjermyn** [MESAHub/mesa#270]
I count six test cases that say their initial models come from `make_co_wd`. Grep output:
```
wd_he_shell_ignition/inlist_he_shell_ignition
9:      load_model_filename = 'co_wd_0.96M.mod' ! from make_co_wd, 13738, initial_mass = 6.0d0

wd_acc_small_dm/inlist_wd_acc_small_dm
9:      load_model_filename = 'co_wd_1.00M.mod' ! from make_co_wd, 13738, initial_mass = 6.4d0

wd_c_core_ignition/inlist_relax_mass
4:      load_model_filename = 'make_co_wd_6M.mod' ! WD from 6Msun (ZAMS) via make_co_wd

wd_stable_h_burn/inlist_wd_stable_h_burn
9:      load_model_filename = 'co_wd_1.00M.mod' ! from make_co_wd, 13738, initial_mass = 6.4d0

gyre_in_mesa_wd/inlist_pulse_wd
7:      load_model_filename = 'co_wd_0.85M.mod' ! from make_co_wd, 13738, initial_mass = 4.7d0

wd_cool_0.6M/inlist_wd_cool_0.6M
11:      ! comes from final.mod produced by make_co_wd as of version 14731
```

For starters we should maybe be a little worried that these haven't been regenerated in a long time. But more importantly, `make_co_wd` isn't run as part of the test suite, so we have no idea if it works or not.

I ran into this on the `Skye_on_PC_off` branch, where `wd_c_core_ignition` is showing a failure that I was worried was due to a starting model generated with a different EOS (I'm now less sure that this is the problem). Along the way I went to try regenerate the starting model using `make_co_wd` and couldn't. At least on that branch on commit `2c865da` it fails on an all-inlist run with the following tail:
```
        366   8.351401  1.629E+04   2.826401   3.238048   0.605815   0.001019   0.000000   0.000000   0.317390  19.734221   1384     21
-3.7561E+00   8.089088  -0.792815   2.891026   2.066940 -99.000000   0.604797   0.000000   0.765029   0.020402   5.317551     12
 4.3220E+08   6.282687   0.216866   2.449357 -99.000000  -8.102218   0.577366   0.208947   0.001936   0.003286  0.000E+00  max increase
                               rel_E_err    6.2863672357018316D-10
                       log_rel_run_E_err       -7.1783317924390762

 k, k_a, k_b         896         896         896
 s%top_conv_bdy(i)= T
 D(k)   2.0421484679788882     
 s%D_mix(k+1)   0.0000000000000000     
 s%overshoot_D_min   100.00000000000000     
 Invalid location for overshoot boundary: cz_bdy_dq, dq=  -8.7406538438728199E-003   1.8221829671880713E-004
terminated evolution: nonzero_ierr

                 runtime (minutes), retries, steps        9.05        21       366
DATE: 2021-05-28
TIME: 09:10:51

******************************************************************
failed to create co_wd.mod when running inlist_remove_env_header
******************************************************************
```

**Comment by adamjermyn** [comment]
I just checked this one on the `TDC_on_default` branch and it works now. No idea why, and unclear if TDC is actually being called (this just happened to be the branch I was working on).

I'll re-enable it in that branch and hopefully it'll keep working once we merge back into main.

**Comment by adamjermyn** [comment]
Resolved with PR #293
